### ISSUE TITLE
Accessibility Update for Claims Steps

### ISSUE THREAD

**Issue by brea11y** [department-of-veterans-affairs/va-mobile-app#8645]
## Description 
During the Section 508 / BVA meeting on 5/15/2024, a screenreader user expressed confusion with the claims process / steps in the app. She began explaining how in her active claim, it said "complete", but there was no letter. We talked about it a bit further and it became clear that she was only on the second or third step in the claims process, but that it wasn't completely clear based on what she was hearing.

Although we are currently in the process of updating the claims area, I recommend that we update the accessibility labels to be more clear to a screenreader user which step is currently active.

Currently, if a step is complete, it reads out as "Step 1 of 5. Completed..." If a step is active, it reads out as "Step 2 of 5. Current..." I'd like to be a little more clear (a "you are here", if you will).

- For a completed step, I would update the label to say: "This step is complete."
- For an active / in-progress step, I would update the label to say: "This step is in-progress."

We could also loop Misty in for recommendations on a way to make the labels super clear with copy.

## Acceptance Criteria
  

-  [ ] Update the accessibility label for completed claim steps
-  [ ] Update the accessibility label for in-progress / active steps

**Comment by wavelaurenrussell** [comment]
Thanks. 'Completed' / 'Current' feels like useless hint text that we can strip away, and just have it read aloud naturally the words on the screen. I'd imagine the actual step name is more useful than assigning Hints in this case.

**Comment by brea11y** [comment]
I agree. It does read the words naturally, but it adds the "Step 1 of X" and the status ("completed" or "current") at the beginning and then it reads everything that is present on the screen. 

One other thing that the user said was that she kept going through the different steps and was listening until the end for VoiceOver to announce which step she was actively on. She expected it to announce the "you are here" at the end and not the beginning, so she didn't even notice the word "current" after the step.

**Comment by wavelaurenrussell** [comment]
Tracking in [8898](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8898)

**Comment by brea11y** [comment]
Since the design of claims is changing, this ticket is no longer valid.
